apps/fulltextsearch.sh

```diff
@@ -48,12 +50,18 @@ else
     done
     # Removal Docker image
     docker_prune_this "$nc_fts"
+    docker_prune_this "$opens_fts"
+    docker_prune_this "esdata"
+    docker_prune_this "fts_os-data"
```

Comment:
is docker_prune_this really able to prune volumes?

Reply:
actually it only stop them, then it prunes everything is not working.
https://github.com/nextcloud/vm/blob/74616ea74ea9ee3fa34ec7875e71d02f94b0805d/lib.sh#L1636-L1647

Reply:
ah okay, fine by me then